src/bot.js

```diff
@@ -132,6 +132,31 @@ function replyToMessages() {
       network.blockAuthorOfMessageWithId(message['id']);
     });
 
+  filterPrivateMessages(messages)
+    .filter(message => message['subject'].match(/refresh (\w+)/i))
+    .forEach(message => {
+      const commentId = message['subject'].match(/refresh (\w+)/i)[1];
+      const comment = network.getComment('t1_' + commentId);
+
+      if (! comment) return;
+
+      const commentReplies = network.getCommentReplies(comment['link_id'], commentId);
```

Comment:
An architecture change idea:

Instead of sending a message with the OP's message id and then searching through the comment replies, we could send a message with the bot's message id and it should have a free `parent_id` property attached to it (...I think). That way we don't need the function `getCommentReplies`, we can just reuse `getComment`

Reply:
I tried this originally and unfortunately got all the way through my implementation before I realised why it isn't suitable: to provide the `refresh conversion` link in the footer of the comment we need to include an ID, and the only ID available to us when replying is the ID of the parent comment, we don't have the bot reply ID until after the bot reply has been made.

There were 2 options: either edit every bot comment immediately after it is made to add the reply ID to the refresh request link, or use the comment replies to identify the bot reply when a refresh request is received.

I chose to go with the second method because it will require fewer API requests and reduces the surface of my changes. The first method would double the API requests the bot does for every conversion. I am open to switching to the edit every conversion comment method though if that's preferable :)

Reply:
LOL, that's a tricky one (but totally makes sense now that you spell it out)! I think you made the right choice, getting child replies sound easier than editing every single comment.